<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="HATI-SDG00096" failures="0" tests="2" name="BillingTest.OP_Patient_Self_Deposit_Debit_Note_Full_Consume_Test" time="10.926" errors="1" timestamp="2024-01-11T00:15:42 IST" skipped="0">
  <testcase classname="BillingTest.OP_Patient_Self_Deposit_Debit_Note_Full_Consume_Test" name="TC_01_Verify_Login" time="10.912"/>
  <system-out/>
  <testcase classname="BillingTest.OP_Patient_Self_Deposit_Debit_Note_Full_Consume_Test" name="TC_02_User_Able_To_Do_Deposit_Consume_Debit_Note" time="0.014">
    <error message="Unresolved compilation problem: 
	The declared package &quot;BillingPages.Depositself&quot; does not match the expected package &quot;BillingPages&quot;
" type="java.lang.Error">
      <![CDATA[java.lang.Error: Unresolved compilation problem: 
The declared package "BillingPages.Depositself" does not match the expected package "BillingPages"

at BillingPages.OP_Patient_Self_Deposit_Debit_Note_Full_Consume_Page.<init>(OP_Patient_Self_Deposit_Debit_Note_Full_Consume_Page.java:1)
at BillingTest.OP_Patient_Self_Deposit_Debit_Note_Full_Consume_Test.TC_02_User_Able_To_Do_Deposit_Consume_Debit_Note(OP_Patient_Self_Deposit_Debit_Note_Full_Consume_Test.java:49)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1207)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- TC_02_User_Able_To_Do_Deposit_Consume_Debit_Note -->
  <system-out/>
</testsuite> <!-- BillingTest.OP_Patient_Self_Deposit_Debit_Note_Full_Consume_Test -->
